controllers/grid/queries/form/QueryForm.inc.php

```diff
@@ -28,7 +28,7 @@ class QueryForm extends Form {
 	/** @var Query The query being edited **/
 	var $_query;
 
-	/** @var boolean True iff this is a newly-created query */
```

Comment:
"iff" is shorthand for "if and only if".

Reply:
@asmecher Done (I recreated the commit)!